floodviz/static/js/map.js

```diff
@@ -56,6 +56,8 @@
 			var maptip = d3.select('body')
 				.append('div')
 				.attr('id', 'maptip');
+			// Google Analytics Boolean Trackers
+			var map_moused_over_gauge = {};
```

Comment:
Nit: Please use 'gage' rather than "gauge". It's the standard spelling for USGS. Let's not rock the boat!